React-native.md

```diff
@@ -0,0 +1,84 @@
+# React-Native 
+
+React Native is a JavaScript framework Allow you to writes real, natively rendering mobile applications for iOS and Android ,based on React.
+You can create code just once and use it to power both their iOS and Android apps. This translates to huge time and resource savings. ,React Native has the ability to render APIs in Objective-C (for iOS) or Java (for Android),your application will render using “bridge” to become real mobile UI components,and will look and feel like any other mobile application,your React Native apps can access platform features like the phone camera, or the user’s location.
```

Comment:
You often need to write different code for Android and iOS. React Native is considered to be "learn once write anywhere", rather than "write once run anywhere".

There are three great things about React Native

1) You do not have to learn both Objective C and Java to write apps for iOS and Android. You only need to know JavaScript and so the same developer can work on both apps. Big win!

2) Some (but not all) of the code is shared between an iOS and Android so this does save time.

3) The development experience is excellent. Javascript and React are powerful tools for building web and mobile applications